deploy/vagrant/conf/ec2.yml

```diff
@@ -6,12 +6,67 @@ Keypair: "aws-east"
 # Path to the ssh key
 Key_Path: "~/.ssh/aws-east.pem"
 
+# Differences among EC2 instance types: http://aws.amazon.com/ec2/instance-types/
+# When using spark, we recommend "r3.large", compile spark needs 2G memory at least
+#
+# Some Instance Type must need PV/HVM(refer to AMI below)
+Instance_Type: "m1.small"
+
+# tag to the instance name to avoid name conflict during mutiple deployment
```

Comment:
mutiple -> multiple

Also, could you make these into complete sentences, with periods.